src/DssSpell.t.sol

```diff
@@ -28,7 +68,112 @@ contract DssSpellTest is DssSpellTestBase {
         assertEq(vat.can(address(pauseProxy), address(daiJoin)), 1);
 
         assertEq(vat.sin(address(vow))         - prevSin,            amountTotal        * RAD);
-        assertEq(dai.balanceOf(MAKERMAN)       - prevDaiMakerMan,    amountMakerMan     * WAD);
+
+        // Recognized Delegates
+        assertEq(dai.balanceOf(FLIPFLOPFLAP)   - prevDaiFlipFlopFlap, amountFlipFlopFlap * WAD);
+        assertEq(dai.balanceOf(FEEDBLACKLOOPS) - prevDaiFeedBlack,    amountFeedBlack    * WAD);
+        assertEq(dai.balanceOf(ULTRASCHUPPI)   - prevDaiUltraSchuppi, amountUltraSchuppi * WAD);
+        assertEq(dai.balanceOf(MAKERMAN)       - prevDaiMakerMan,     amountMakerMan     * WAD);
+        assertEq(dai.balanceOf(ACREINVEST)     - prevDaiAcreInvest,   amountAcreInvest   * WAD);
+        assertEq(dai.balanceOf(MONETSUPPLY)    - prevDaiMonetSupply,  amountMonetSupply  * WAD);
+        assertEq(dai.balanceOf(JUSTINCASE)     - prevDaiJustinCase,   amountJustinCase   * WAD);
+        assertEq(dai.balanceOf(GFXLABS)        - prevDaiGfxLabs,      amountGfxLabs      * WAD);
+
+        // Core Units
+        assertEq(dai.balanceOf(CES_WALLET)     - prevDaiCES,         amountCES          * WAD);
+        assertEq(dai.balanceOf(IS_WALLET)      - prevDaiIS,          amountIS           * WAD);
+    }
+
+
+    uint256 constant MAR_01_2022            = 1646092800; // 2022-03-01
+    uint256 constant APR_01_2022            = 1648771200; // 2022-04-01
+    function testDaiStreams() public {
+        uint256 streams = vestDai.ids();
```

Comment:
I'd put a check here proving that the actual number of streams is 29. So we can say that the new ones are 30, 31 and 32.